armory/utils/config_loading.py

```diff
@@ -136,10 +138,26 @@ def load_attack(attack_config, classifier):
         if targeted:
             logger.warning("Patch attack generation may ignore 'targeted' set to True")
         attack_config["kwargs"] = kwargs
+    else:
+        if attack_config.get("type") not in SUPPORTED_TYPES:
+            logger.warning(
```

Comment:
To avoid a warning for users with the default configs (in the future), we could also pull out the ```config["attack"]["type"]``` field from scenario_configs/gtsrb_scenario_clbd.json and from scenario_configs/gtsrb_scenario_clbd_defended.json.

Reply:
agreed, done